theatre/studio/src/panels/SequenceEditorPanel/GraphEditor/BasicKeyframedTrack/KeyframeEditor/GraphEditorDotNonScalar.tsx

```diff
@@ -64,7 +65,18 @@ const GraphEditorDotNonScalar: React.VFC<IProps> = (props) => {
 
   const curValue = props.which === 'left' ? 0 : 1
 
-  const isDragging = useDragKeyframe(node, props)
+  const [inlineEditorPopover, openEditor, _, _isInlineEditorPopoverOpen] =
+    useSingleKeyframeInlineEditorPopover({
+      keyframe: props.keyframe,
+      pathToProp: props.pathToProp,
+      propConf: props.propConfig,
+      sheetObject: props.sheetObject,
+      trackId: props.trackId,
+    })
+
+  const isDragging = useDragKeyframe(node, props, (event) =>
+    openEditor(event, event.target instanceof Element ? event.target : node!),
```

Comment:
nit: add a comment explaining why this is called in an event callback on useDragKeyframe. I think its because the drag handler blocks click events right?

Reply:
Yes, good call out, @vezwork. I'll add a comment inline for this

Reply:
Addressed